projects/assets/src/translations/en/my-account.ts

```diff
@@ -16,7 +16,7 @@ export const myAccount = {
     statusDisplay_error: 'Pending',
     statusDisplay_Error: 'Pending',
     statusDisplay_open: 'Open',
-    statusDisplay_processing: 'In Process',
```

Comment:
I think we shouldn't change translations after 1.0.
I'm also not sure, whether this ticket wasn't rather about investigating modelt backend - why it doesn't process orders. FYI @Xymmer